src/mapproxyUtils/mapproxySeed.ts

```diff
@@ -242,17 +242,66 @@ export class MapproxySeed {
         this.logger.info('requested to skip uncached tiles');
         flags.push('--skip-uncached');
       }
+
       const cmd = $`mapproxy-seed ${flags}`;
+      // promise wrap to synchronized zx internal events with node run time event
+      await new Promise<void>((resolve, reject) => {
+        cmd.stdout
+          .on('data', (chunk: Buffer) => {
+            try {
+              this.seedProgressFunc(chunk);
+            } catch (error) {
+              // this section catching seeding errors and fail the task by kill child process of cmd
+              void cmd.kill('SIGHUP');
+              cmd.child.on('exit', () => {
+                reject(new Error((error as Error).message));
```

Comment:
can't we log the error here? (with jobId, ...)

Reply:
already added on line 255